README.md

```diff
@@ -6,9 +6,9 @@
 A JavaScript project for the Leaderboard list app, using webpack and ES6 features, that consumes the Leaderboard API using JavaScript async and await.
 
```

Comment:
- [ ] This module requires that your _ReadMe_ looks professional as much as possible, therefore kindly include the `Getting Started` section to _ReadMe_, which should include instructions on how to **setup** and **run** your application. You can follow this [Micorverse recommended template](https://github.com/microverseinc/readme-template/blob/master/README.md).

Reply:
Hello, @rloterh. Thank you for the professional review. I am grateful for the detailed review report and I have since enforced all of the changes you required.

Reply:
Welcome bro!🙏 😃